### ISSUE TITLE
[HTTP Polling] Bug with multiple users : unique actor

### ISSUE THREAD

**Issue by mxbossard** [gatling/gatling#3034]
Hi,

II use a HTTP Poller to send ping requests. With only one user it works great. I am able to start the poller, stop it on logout, and restart it on next login.
But when a I increase the number of concurrent user (2 for exemple) I got this error : 
`akka.actor.InvalidActorNameException: actor name [gatling.http.polling-383-actor] is not unique!
	at akka.actor.dungeon.ChildrenContainer$NormalChildrenContainer.reserve(ChildrenContainer.scala:129)
	at akka.actor.dungeon.Children$class.reserveChild(Children.scala:130)
	at akka.actor.ActorCell.reserveChild(ActorCell.scala:374)
	at akka.actor.dungeon.Children$class.makeChild(Children.scala:268)
	at akka.actor.dungeon.Children$class.attachChild(Children.scala:46)
	at akka.actor.ActorCell.attachChild(ActorCell.scala:374)
	at akka.actor.ActorSystemImpl.actorOf(ActorSystem.scala:589)
	at io.gatling.http.action.async.polling.PollingStart.startPolling$1(PollingStart.scala:55)`

It seems that only one actor is created by poller name, and it's limitating. 

Plus, it could be great to name a poller with an Expression[String] which could solve the problem.

**Comment by slandelle** [comment]
Sorry, but I can't reproduce.
Poller actor name is based on the virtual user id, so I fail to see how 2 different users could share the same.

Could you please provide a reproducer? Are you sure you don't start the same poller several times per virtual user.


Here's my sample:

```scala
val httpConf = http
  .baseURL("http://gatling.io")
  .acceptHeader("*/*")
  .acceptLanguageHeader("en-US,en;q=0.8")
  .acceptEncodingHeader("gzip, deflate")
  .userAgentHeader("Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.101 Safari/537.36")
  .disableCaching

val scn = scenario("BeautifulPolling")
  .exec(polling
    .every(1 seconds)
    .exec(http("root").get("/")))
  .pause(5)
  .exec(polling.stop)

setUp(scn.inject(atOnceUsers(3))).protocols(httpConf)
```

**Comment by slandelle** [comment]
Note: I have no issue either when forcing the poller name: `.pollerName("heartbeatPolling")`.

**Comment by mxbossard** [comment]
The fact is I start several time the same poller after stopping it. Can I do that ? Maybe the poller is not stoped beacause of a bad handled failure.
Plus, I still use the 2.2.0 version.

**Comment by slandelle** [comment]
I tried the following sample and can't reproduce:

```scala
val scn = scenario("BeautifulPolling")
  .exec(polling
    .every(1 seconds)
    .exec(http("root").get("/")))
  .pause(5)
  .exec(polling.stop)
  .exec(polling
    .every(1 seconds)
    .exec(http("root").get("/")))
  .pause(5)
  .exec(polling.stop)

setUp(scn.inject(atOnceUsers(3))).protocols(httpConf)
```

Could you share a reproducer please?

**Comment by slandelle** [comment]
This is a duplicate of #2970 that was fixed in 2.2.1

**Comment by mxbossard** [comment]
Ok, I'll check that. Thanks !
